board/piksiv3/rootfs-overlay/etc/init.d/S86skylark0_setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+name="skylark_setup"
+
+start() {
+  # Setup DNS
+  # TODO (mookerji): Move to a template command?
```

Comment:
TODO: Resolve! Where should this stuff live? Should the DNS stuff be elsewhere? is this UUID hack ok for now?